ci_framework/roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -29,6 +29,20 @@
     _hosts: "{{ _hosts|default([]) + [ hostvars[item][ceph_hostname_var] ] }}"
   loop: "{{ groups['edpm'] }}"
 
+- name: Get and save TLS certificate for rgw spec if provided
+  when: cifmw_cephadm_certificate | length > 0 and
+        cifmw_cephadm_key | length > 0
+  become: true
+  block:
+    - name: Get the certificate content
+      register: slurp_cert
+      ansible.builtin.slurp:
```

Comment:
`slurp` assumes you're reading a remote file: is that the case? I see the playbook is executed on `edpm[0]`, so we can simple use [1] to read the file content.
On the other side, I'm wondering what's happen when we have 3 different nodes w/ 3 diff certs: do you see any problem with this config?

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_lookup.html